### PR TITLE
Resources v2 blog post

### PR DESCRIPTION
**Pull request by AnuiDolphin**
This is the blog post announcing the updated Resources page for Insights 2.0

### CODE REVIEW

File: content/blog/insights-resources-v2/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: "Updated Resources View"
```
Comment (tyler-dunkel) [inline content/blog/insights-resources-v2/index.md:2]: title seems a little bland. Maybe something like:

`introducing Resources view V2`

File: content/blog/insights-resources-v2/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+* **Filtering** allows you to quickly find specific resources or groups of resources based on different attributes of the resource. Filtering can be done by selecting the filter icon in the column header for the column you want to filter by. This will present a drop down that enables you to select, or enter, the values to filter by. Adding column filters updates the Resource Search field to reflect the filters selected, in this way, the column filtering can be used to help create Resource Queries that can be reused in other contexts.
+* **Sorting** is done in the standard way by clicking on the column header.
+
+{{< video title="Grouping, Filtering, Sorting" src="grouping-sorting-filtering.mp4" autoplay="true" loop="true" >}}
+
+The updated Resources page enables you to customize the view of your resources by enabling you to control the columns of the table. With the table actions menu you can add and remove columns as well as auto size each of the columns in the table to fit the current content. Like previous versions of the Resources page, this menu also enables you to export the Resources as a CSV file for integration into external tools.
```
Comment (tyler-dunkel) [inline content/blog/insights-resources-v2/index.md:25]: maybe add a heading before each of these little paragraphs that have their own video?

**Comment by tyler-dunkel** [comment]
overall I think this looks great!

File: content/blog/insights-resources-v2/index.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+<!--more-->
+
+* **Grouping** provides a rich model for factoring your resources into logical groupings to help you gain insights over your infrastructure and plan actions in incremental groupings. To create a group, drag the column header to the grouping control. Note that you can create hierarchical groups by adding multiple columns and even rearrange the order of the groups.
+* **Filtering** allows you to quickly find specific resources or groups of resources based on different attributes of the resource. Filtering can be done by selecting the filter icon in the column header for the column you want to filter by. This will present a drop down that enables you to select, or enter, the values to filter by. Adding column filters updates the Resource Search field to reflect the filters selected, in this way, the column filtering can be used to help create Resource Queries that can be reused in other contexts.
```
Comment (lblackstone) [inline content/blog/insights-resources-v2/index.md:20]:
```suggestion
* **Filters** - Columns provide type-specific filters to narrow your search. Filters work in conjunction with the free-form search bar, giving you the control to build focused views for different aspects of your infrastructure.
```

File: content/blog/insights-resources-v2/index.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+* **Sorting** is done in the standard way by clicking on the column header.
```
Comment (lblackstone) [inline content/blog/insights-resources-v2/index.md:21]:
```suggestion
* **Sorting** - Each column supports sorting, and multi-sort support allows multiple columns to be included in the ordering.
* **Favorites** - Once you find a view that you like, save it as a favorite to give you quick access to it in the future. The explorer state is persisted to the URL, so you can share the view with other users in your organization.
```

File: content/blog/insights-resources-v2/index.md (hunk 1 above)
Comment (AnuiDolphin) [reply content/blog/insights-resources-v2/index.md:2]: Changed to 'Introducing the new Resources view'

File: content/blog/insights-resources-v2/index.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+meta_image: meta.png
+authors:
+    - craig-symonds
+tags:
+    - insights
+    - resources
+---
+
+Pulumi Insights gives you the tools to stay informed about your cloud infrastructure. Our Resource explorer provides advanced search and filtering to [find what you need](/blog/resource-search). Today, we are excited to release an update that adds new ways to factor your resource data, and share those views with other users in your organization!
```
Comment (beneficialmedia) [inline content/blog/insights-resources-v2/index.md:15]: and filtering too

File: content/blog/insights-resources-v2/index.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+<!--more-->
+
+### Factoring the Data
+
+Being able find and group your resources using diffent fields and properties is the key to gaining the insights needed to understand the status of your infrastructure and determine any future projects needed. The new Resources view provides several updated tools for doing this factoring.
```
Comment (beneficialmedia) [inline content/blog/insights-resources-v2/index.md:21]: different

File: content/blog/insights-resources-v2/index.md (hunk 5 above)
Comment (AnuiDolphin) [reply content/blog/insights-resources-v2/index.md:15]: The comment references filtering - what change are you proposing? "Resource explorer provides advanced search and filtering to [find what you need](/blog/resource-search)."

Comment (AnuiDolphin) [reply content/blog/insights-resources-v2/index.md:15]: Fixed

File: content/blog/insights-resources-v2/index.md (hunk 6 above)
Comment (AnuiDolphin) [reply content/blog/insights-resources-v2/index.md:21]: Fixed

File: content/blog/insights-resources-v2/index.md (hunk 2 above)
Comment (AnuiDolphin) [reply content/blog/insights-resources-v2/index.md:31]: Fixed - headers added.

**Review by tyler-dunkel — APPROVED** [review]

**Pull request merged**